index.html

```diff
@@ -15,11 +15,16 @@
 <body>
   <header class="presentation">
     <nav class="nav-container">
-      <img class="my-logo" src="Images/Logo HG.png">
-      <i style="color: #acb7c3" class="fa fa-bars"></i>
+      <img class="my-logo logo-animation" src="Images/Logo HG.png">
+      <i style="color: #acb7c3" class="fa fa-bars desktop-hide"></i>
+      <ul class="mobile-hide nav-items-list">
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
```

Comment:
- [ ] [OPTIONAL] You will agree with me that these are the navigation elements of your portfolio which  were supposed to allow visitors to have a good navigation experience as they explore your page. But this is not possible because you have not used a `a` tag to indicate this powerful accessibility tool. As a convention, links are wrapped in `<a href="#link here>Link Text</a>` tags. Consider wrapping that in your `ul`/`li` tags. :hammer_and_wrench: